os_migrate/roles/conversion_host/tasks/unregister.yml

```diff
@@ -6,5 +6,6 @@
     ansible_become: true
     ansible_become_user: root
   when:
+    - os_migrate_conversion_rhsm_manage and
```

Comment:
I think the `and` at the end will break it?

Reply:
These are separate list items with an implicit `and` operation between them.

Reply:
right, sorry I copied that from the other condition and didn't remove the and this will fails for sure